thirdparty/build-thirdparty.sh

```diff
@@ -1239,6 +1239,20 @@ build_clucene() {
     fi
 }
 
+# avx2ki
+build_avx2ki() {
+    check_if_source_exist $AVX2KI_SOURCE
+    cd $TP_SOURCE_DIR/$AVX2KI_SOURCE
+    rpm2cpio boostkit-ksl-2.0.0-1.aarch64.rpm | cpio -id
```

Comment:
which platform will this library for, can it work on both centos7 and ubuntu22.04?